libtransmission/torrent-magnet.h

```diff
@@ -39,7 +39,12 @@ class tr_metadata_download
         return size > 0 && size <= std::numeric_limits<int>::max();
     }
 
-    bool set_metadata_piece(int64_t piece, void const* data, size_t len);
+    [[nodiscard]] constexpr auto is_complete() const noexcept
```

Comment:
I don't think `std::deque<>::empty()` is `constexpr` in the standard?

Reply:
48e7001